packages/atlas/src/hooks/useReactionTransactions.ts

```diff
@@ -87,28 +109,54 @@ export const useReactionTransactions = () => {
           ),
         onTxSync: async ({ block }) => {
           const refetchResult = await refetchComments()
-          setCommentInputProcessing(false)
+          setCommentInputIsProcessing({ commentInputId, value: false })
 
-          const newCommentsQueryResult = refetchResult[0] as GetCommentsConnectionQueryHookResult
+          const newCommentsQueryResult = refetchResult[0]
           // TODO - We probably shouldn't use inBlock here - it's possible that we could create multiple comments in one block
           // Update once https://github.com/Joystream/atlas/issues/2629 is done.
-          newCommentId = newCommentsQueryResult.data?.commentsConnection.edges.find(
-            (edge) => edge.node.commentcreatedeventcomment?.[0].inBlock === block
-          )?.node.id
+          newCommentId = newCommentsQueryResult.data?.userComments.find(
```

Comment:
Actually, we should use `videoCommentsConnection` here,  the userComments query ignores comments which are replies to another comment.

Reply:
fixed!